packages/webview_flutter/webview_flutter_android/pigeons/android_webview.dart

```diff
@@ -57,6 +57,48 @@ enum FileChooserMode {
   save,
 }
 
+// TODO(bparrishMines): Enums need be wrapped in a data class because thay can't
+// be used as primitive arguments. See https://github.com/flutter/flutter/issues/87307
+class FileChooserModeEnumData {
+  late FileChooserMode value;
+}
+
```

Comment:
It looks like this was added by accident. I couldn't find anywhere this is used.

```suggestion
```